docs/src/main/paradox/scala/http/release-notes.md

```diff
@@ -5,7 +5,25 @@
 ### Support for HTTP(S) proxies with Authorization
 
 It is now possible to connect to @scala[@ref[HTTP(S) Proxies](client-side/client-transport.md)]@java[@ref[HTTP(S) Proxies](client-side/client-transport.md)]
-that require an authorization via an `Proxy-Authorization` header. This can be set up directly on the `ClientTransport` object when configuring the proxy.
+that require an authorization via an `Proxy-Authorization` header. This can be set up directly on the `ClientTransport` object when configuring the proxy. ([#1213](https://github.com/akka/akka-http/issues/1213))
+
+### Improvements
+
+* (server) Better error reporting when failing to handle CONNECT requests ([#1315](https://github.com/akka/akka-http/issues/1315))
+* Add HttpApp.startServer(host, port, system) ([#1294](https://github.com/akka/akka-http/issues/1294))
+* Logging with println in Http2ServerDemux ([#1275](https://github.com/akka/akka-http/issues/1275))
```

Comment:
I guess that was rather a bugfix. Let's make sure that all lines are phrased in the same way, i.e. not stating the bug but the fix. In this case e.g. it's "Remove println logging in Http2 implementation".